Jenkinsfile

```diff
@@ -174,22 +174,14 @@ stage('Build') {
 									sh "./docker_db.sh postgresql"
 									state[buildEnv.tag]['containerName'] = "postgres"
 									break;
-								case "postgresql_10":
-									// use the postgis image to enable the PGSQL GIS (spatial) extension
-									docker.withRegistry('https://index.docker.io/v1/', 'hibernateci.hub.docker.com') {
-										docker.image('postgis/postgis:10-2.5').pull()
-									}
-									sh "./docker_db.sh postgresql_10"
-									state[buildEnv.tag]['containerName'] = "postgres"
-									break;
 								case "edb":
 									docker.image('quay.io/enterprisedb/edb-postgres-advanced:15.2-3.3-postgis').pull()
 									sh "./docker_db.sh edb"
 									state[buildEnv.tag]['containerName'] = "edb"
 									break;
-								case "edb_10":
-									docker.image('quay.io/enterprisedb/edb-postgres-advanced:10.22').pull()
-									sh "./docker_db.sh edb_10"
+								case "edb_11":
```

Comment:
Remove this entry from this file please.

Reply:
I'm not sure I understand? remove the edb_11?